node.js/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "logdna_cloudwatch",
+  "version": "1.0.0",
+  "description": "Lambda Functions to Stream Logs from AWS CloudWatch to LogDNA",
+  "main": "index.js",
+  "dependencies": {
```

Comment:
Do we need `zlib`?

Reply:
Yes, certainly but it's built-in - no need to include it into `package.json`